src/dapps/preview/preview-config.ts

```diff
@@ -26,6 +26,7 @@ export type PreviewConfig = {
   projection: PreviewProjection
   autoRotateSpeed: number
   centerBoundingBox: boolean
+  fadeEffect: boolean
   offsetX: number
```

Comment:
Should `peerUrl` and `nftServerUrl` also be added here?

Reply:
They are not actually needed in the `PreviewConfig`. There are two concepts that are not very obvious at a glance, which are the `PreviewOptions` and the `PreviewConfig`. The former is what the users configures (either via query params or postMessage) and the latter is what the renderer needs to work. The options could be a profile, an array of URNs, some blobs, and the config would be an array of `Wearable[]` already loaded in memory. So the webapp takes the options, resolves and fetches all the resources needed, and generates the config, which is then feed to the renderer. The `peerUrl` and `nftServerUrl` are needed in the process to generate the config, but not within the renderer itsef, so no need to add it to the `PreviewConfig`.